From b9b11a8bb34fa5903795cd10f598fdb5e65aa0bb Mon Sep 17 00:00:00 2001
From: John Ullyses Capellan <john.capellan@mdscsi.com>
Date: Thu, 11 Aug 2022 11:07:35 +0800
Subject: [PATCH] main

---
 index.php | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/index.php b/index.php
index feb9cfd..32caa6c 100644
--- a/index.php
+++ b/index.php
@@ -3,6 +3,6 @@
   <title>PHP Test</title>
  </head>
  <body>
- <?php echo '<p>Hello World</p>'; ?> 
+ <?php echo '<p>Hello Main</p>'; ?> 
  </body>
 </html>
\ No newline at end of file
-- 
2.36.0

